### PR TITLE
Update vertex color calculation for metal-roughness and spec-gloss

### PR DESCRIPTION
**Pull request by sbtron**
Adding explanation on how pbr materials properties interact with vertex colors if they are present.

### CODE REVIEW

File: extensions/Khronos/KHR_materials_pbrSpecularGlossiness/README.md [hunk 1]
```diff
@@ -66,9 +66,9 @@ The specular property from specular-glossiness material model is the same as the
...
 
-The following equations show how to calculate bidirectional reflectance distribution function (BRDF) inputs (*c<sub>diff</sub>*, *F<sub>0</sub>*, *&alpha;*) from the metallic-roughness material properties. 
+The following equations show how to calculate bidirectional reflectance distribution function (BRDF) inputs (*c<sub>diff</sub>*, *F<sub>0</sub>*, *&alpha;*) from the metallic-roughness material properties. In addition to the material properties, if a primitive specifies a vertex color using the attribute semantic property `COLOR_0`, then this value acts as an additional multiplier to *c<sub>diff</sub>*.
 
-*c<sub>diff</sub>* = `diffuse.rgb * (1 - max(specular.r, specular.g, specular.b)`
+*c<sub>diff</sub>* = `diffuse.rgb * (1 - max(specular.r, specular.g, specular.b) * vertexColor`
```
Comment (javagl) [inline extensions/Khronos/KHR_materials_pbrSpecularGlossiness/README.md]: Missing ) here?

**Comment by peteroupc** [comment]
Can you explain why the vertex color appears in the calculation as given here?  Is this how common physically-based rendering engines treat vertex colors?

**Comment by bghgary** [comment]
The original thought was that we wanted vertex colors to affect metallic-roughness the same as specular-glossiness PBR models. But it looks like at least BabylonJS and three.js both multiply against the baseColor of the metallic-roughness material instead of what is in this PR. We will likely update the PR to reflect this.

**Comment by pjcozzi** [comment]
@sbtron did you make the spec update for this?

**Comment by sbtron** [comment]
@pjcozzi finally updated if you want to review.

**Comment by pjcozzi** [comment]
LGTM

**Pull request merged**